apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/IContextWrapper.cs

```diff
@@ -12,23 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Microsoft.AspNetCore.Http;
-using System;
-using System.Threading.Tasks;
-
-namespace Google.Cloud.Diagnostics.AspNetCore
+namespace Google.Cloud.Diagnostics.Common
 {
     /// <summary>
-    /// A generic exception logger.
+    /// Wraps an http context.
     /// </summary>
-    public interface IExceptionLogger
+    internal interface IContextWrapper
     {
         /// <summary>
-        /// Asynchronously logs an exception that occurred.
+        /// Gets the http method.  GET, POST, ect. Can be null.
+        /// </summary>
+        string GetMethod();
```

Comment:
It may be better to name this GetHttpMethod since "method" has other connotations.